app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V1::UsersController < ApplicationController
+  before_action :find_user, except: %i[index]
```

Comment:
- [ ] Nice working on your project, kindly check I get the error in the screenshot below when trying to login using postman.
![image](https://user-images.githubusercontent.com/48371156/206825279-9336dc3c-a1ed-4841-9c0d-199588b96ef3.png)